fuzz/fuzz_targets/fuzz.rs

```diff
@@ -334,6 +334,13 @@ impl<const D: usize> Workload<D> {
             assert!(!intersects);
         }
 
+        // HINT: uncomment this temporarily to test with small BVH's.
+        /*
+        if !self.mutations.is_empty() || self.shapes.len() > 5 {
+            return;
+        }
+        */
```

Comment:
Hm, that's a bit weird to keep it here. Can we find a solution where we'll benefit from this permanently and not just when it's commented in? You can also keep it this way if you think it's actually helpful like this.

Reply:
The idea is that we want to detect bugs in large BVH's (or bugs from mutations). Once a bug is detected, it **may** be reproducible with a simpler BVH. I find myself adding this sort of thing while debugging almost every time.

I have an idea, though. We could run two instances of the fuzzer in parallel, with one version checking all BVH's and one checking small ones. Then, the output of the small BVH fuzzer may show simple issues, without the risk of missing issues that only affect large BVH.